### PR TITLE
remove this in static function

### LINKED ISSUE #1146: Master Version / no save button on Licenses
Hello

I test the new license module and I don't have save button to save changes.

It would be useful to add a column group in user licenses view

Thank you for your great job on GLPI 9.1 !!!

image

### PR DESCRIPTION
**Pull request by orthagh**
fix #1146
Since fb4b66b

### CODE REVIEW

**Comment by ddurieux** [comment]
Add a test please ;)

File: inc/alert.class.php [hunk 1]
```diff
@@ -231,11 +231,11 @@ static function displayLastAlert($itemtype, $items_id) {
...
-                                                  'ORDER'    => 'date DESC',
-                                                  'LIMIT'    => 1,
-                                                  'itemtype' => $itemtype,
-                                                  'items_id' => $items_id]);
+         $iter = $DB->request(getTableForItemType(__CLASS__), ['FIELDS'   => 'date',
+                                                               'ORDER'    => 'date DESC',
+                                                               'LIMIT'    => 1,
+                                                               'itemtype' => $itemtype,
+                                                               'items_id' => $items_id]);
          if ($row = $iter->next()) {
```
Comment (remicollet) [inline inc/alert.class.php]: self::getTable() seeems better ;)

Comment (orthagh) [inline inc/alert.class.php]: Sure, i copypasted from the original commit.

**Comment by remicollet** [comment]
Merged, thanks.

**Pull request closed without merging**